src/pages/patientView/trialMatch/TrialMatchTable.tsx

```diff
@@ -577,19 +575,36 @@ export default class TrialMatchTable extends React.Component<ITrialMatchProps> {
     public getDescriptionForNotMatches(
         genomicAlteration: string[],
         threshold: number,
-        preContent: string,
-        postContent: string
+        type: string
     ) {
         const hugoSymbolSet = new Set(
             [...genomicAlteration].map((s: string) => s.split(' ')[0])
         );
         let genomicAlterationContent = '';
-        if (hugoSymbolSet.size <= threshold) {
-            genomicAlterationContent = [...hugoSymbolSet].join(', ');
-        } else {
-            genomicAlterationContent = `${hugoSymbolSet.size} genes`;
+        if (type === 'mutation') {
```

Comment:
you should make an TS enum for these type values

Reply:
Thanks for pointing it out. I have created enum type for alteration types.